### ISSUE TITLE
popHealth Installation with latest  pophealth code and the 2.5.0 measure bundle from September

### ISSUE THREAD

**Issue by dabimahesh** [OSEHRA/popHealth#1]
I have followed the latest installation instructions to the letter, and I am running into the #39 issue with delayed job execution - failed with error 16722: "exception: SyntaxError: Unexpected token ;"

Downgrading fixed the issue, so I believe this is a problem somewhere between the current popHealth source code, the gem versions it uses, and the version of the measures bundle that gets imported.

Here is what fixed the issue for me:

git reset --hard eb00169578
delete all gems and do bundle install again
drop all dbs in mongo
download an older bundle (2.4.0): curl -u NLM_USERNAME:NLM_PASSWORD http://demo.projectcypress.org/bundles/bundle-2.4.0.zip -o ../bundle-2.4.0.zip
re-import this older bundle: bundle exec rake bundle:import[../bundle-2.4.0.zip,false,false,'*',true,true]
Please try installing using the latest instructions and you are guaranteed to reproduce this error.

**Comment by petercyli** [comment]
Thanks for the letting the group now.  We will reproduce this error and add it to the list of issues to work on for our next release.

**Comment by zohairhussain** [comment]
Thanks Mahesh. will try this and let the group know.

**Comment by dabimahesh** [comment]
In the latest code when I import bundle 2.6, everything works fine but somehow after few mins machine becomes very slow. The only option to make it responsive is to restart. Any idea what might be the issue?

Thanks
Mahesh

**Comment by smc-ssiddiqui** [comment]
Thanks for the update Mahesh. Is anybody else seeing the same issue with  bundle 2.6.0?

**Comment by cporichie** [comment]
I am having the slowness issue as well but it is taking longer than a few minutes. About 24 hours later my VM is so slow I can't open anything. Resources look OK except the ram is high, but not maxed. I am going to see what I can find out, but thought I would report it here as well.

**Comment by petercyli** [comment]
Can you use "top" command, and then shift+m to see what are the process using all of the memory?  Attached the screen shot image so we can see the allocation of memory.
Thanks.

**Comment by cporichie** [comment]
This image is shortly after a reboot since the server locked up again.

3022 is slowly growing memory and is the jobs:work run manually in the terminal. The server has no users on the web and all patients have already been summarized on a previous run. 3115 released its memory while I typed this. CMS 117 was intentionally left out due to known issues.

![image](https://cloud.githubusercontent.com/assets/10074696/9606417/5726f62c-5091-11e5-9945-719afaecbfab.png)

**Comment by cporichie** [comment]
Standby let me give this thing 8 gigs and see what happens later today. It has 1-8 dynamic, but maybe it doesn't like that.

**Comment by smc-ssiddiqui** [comment]
What version of Mongo is everyone using? We have seen memory issues with v3. So far it looks like version 2.4 is working well even with larger data loads (up to 40k patients)

**Comment by cporichie** [comment]
2.6.11
My instance is straight from this wiki.
https://github.com/OSEHRA/popHealth/wiki/Installation

**Comment by smc-ssiddiqui** [comment]
This is a recently found issue. We will update the documentation. Please let us know how things go with increased RAM. Thanks,

**Comment by cporichie** [comment]
Hi, sorry for the delay on this. With the extra ram the machine lasted quite a bit longer, but I became sidetracked with other work. From early Sept until now I hadn't used the machine and when I connect I see that the worker job is using 70% of the ram even though it hasn't done any work in over a month. There must be a leak or missing release command (I am new to ruby so forgive me). Maybe even an incompatibility between the different dependencies. I am unable to ctrl-C the terminal where it runs. I then tried killing and it would not kill. I had to kill all ruby processes to make it go away.
